common/v2/services/EthService/utils/signing.ts

```diff
@@ -41,7 +41,7 @@ export function verifySignedMessage({ address, msg, sig, version }: ISignedMessa
   }
   //TODO: explain what's going on here
   sigb[64] = sigb[64] === 0 || sigb[64] === 1 ? sigb[64] + 27 : sigb[64];
-  const hash = version === '2' ? hashPersonalMessage(toBuffer(msg)) : sha3(msg);
+  const hash = version === '2' ? hashPersonalMessage(toBuffer(msg)) : keccak(msg);
```

Comment:
Will this allow us to still verify messages signed with `sha3` instead of `keccak`? If this was a recent change, we'll want to support that backwards compatibility, if it's not too much effort.

Reply:
They just remove an export in newer version, functionality is still the same.
![image](https://user-images.githubusercontent.com/12676806/74256470-ae1b2580-4cf3-11ea-85c8-049adc9b3e20.png)

Reply:
aye. gotcha. thanks c: